products/freebsd.md

```diff
@@ -146,11 +146,11 @@ releases:
 
 ---
 
-> The [FreeBSD](https://www.freebsd.org/) operating system powers modern servers, desktops, and embedded platforms.
+> [FreeBSD](https://www.freebsd.org/) is an operating system used to power modern servers, desktops, and embedded platforms.
 
-[The support model](https://www.freebsd.org/security/#sup), summarised:
+Under the [current support model](https://www.freebsd.org/security/#sup), each major version’s stable branch
```

Comment:
No need to mention that the model is _current_. It was established more than seven years ago, and is not likely to change in the foreseeable future.

Reply:
You are right, fixed.